src/controlGroups/Knob.tsx

```diff
@@ -57,13 +58,13 @@ const Knob = memo((props: KnobProps) => {
         endAngle
       );
 
-      const currentValue = degreesToParam(
-        paramMin,
-        paramMax,
```

Comment:
Now that I've unified the mapping functions into a single function, I've just changed the inputs around to do the reverse map.